lib/core/utils/preload.js

```diff
@@ -66,7 +66,7 @@ axe.utils.getPreloadConfig = function getPreloadConfig(options) {
 	if (
 		options.preload.timeout &&
 		typeof options.preload.timeout === 'number' &&
-		!Number.isNaN(options.preload.timeout)
+		!isNaN(options.preload.timeout)
```

Comment:
`Number.isNaN` does not exist in IE11, and was causing the timeout test to fail, but not because of the timeout